src/config/backends/index.ts

```diff
@@ -3,9 +3,20 @@ import { ExtraUEnv } from './extra-uEnv';
 import { ConfigTxt } from './config-txt';
 import { ConfigFs } from './config-fs';
 
-export default [
+export const all = [
```

Comment:
```suggestion
export const allBackends = [
```

Reply:
I was trying to avoid being too verbose. Ie import `backend` module and using `allBackends` variable has backend twice but I am not entirely convinced being generic is *better*...I'll revert to being more specific.